packages/common-scripts/index.d.ts

```diff
@@ -1,7 +1,9 @@
-import { TransactionSkeleton } from "@ckb-lumos/types"
+import { TransactionSkeleton } from "@ckb-lumos/helpers"
 
 export type Address = string
 export type Config = any // TODO: define this type later
+export type FullCell = any
+export type CellProvider = any
```

Comment:
`CellProvider` can also be defined now:

1. Let's define a `QueryOptions` structure that might take lock, type, typeIsNull and data.
2. A `CellProvider` is just one type with a `collector` method that takes a `QueryOptions`, and emits a [CellCollector](https://github.com/xxuejie/ckb-js-toolkit/blob/master/index.d.ts#L132)

Reply:
updated